core/modules/file/file.module

```diff
@@ -4,9 +4,92 @@
  * Defines a "managed_file" Form API field and a "file" field for Field module.
  */
 
+/**
+ * Modules should return this value from hook_file_entity_access() to allow
+ * access to a file.
+ */
+define('FILE_ACCESS_ALLOW', 'allow');
+
+/**
+ * Modules should return this value from hook_file_entity_access() to deny
+ * access to a file.
+ */
+define('FILE_ACCESS_DENY', 'deny');
+
+/**
+ * Modules should return this value from hook_file_entity_access() to not affect
+ * file access.
+ */
+define('FILE_ACCESS_IGNORE', NULL);
+
+/**
+ * The {file_managed}.type value when the file type has not yet been determined.
+ */
+define('FILE_TYPE_NONE', 'undefined');
+
+/**
+ * File types constant for user-defined file types.
+ */
+define('FILE_TYPE_STORAGE_NORMAL', 1);
+
+/**
+ * File type constant for file types that override module-defined presets.
+ */
+define('FILE_TYPE_STORAGE_OVERRIDE', 2);
+
+/**
+ * File type constant for module-defined file types.
+ */
+define('FILE_TYPE_STORAGE_DEFAULT', 4);
+
 // Load all Field module hooks for File.
 require_once BACKDROP_ROOT . '/core/modules/file/file.field.inc';
 
+/**
+ * Implements hook_hook_info().
+ */
+function file_hook_info() {
+  $hooks = array(
+    // File API hooks
+    'file_copy',
+    'file_move',
+    'file_validate',
+    'file_type_info',
+    'file_type_info_alter',
```

Comment:
I can't find any documentation or usage of any of these above hooks, e.g. `hook_file_copy()`, `hook_file_move()`, `hook_file_validate()`, `hook_file_type_info()`, and `hook_file_type_info_alter()`. Do these hooks actually exist?

Reply:
Oh I see `hook_file_copy/move/validate()` are existing hooks that aren't modified in this PR. Where's `file_type_info()` used though?

Reply:
I removed the hooks we don't have anymore from this list.

Reply:
...and those comments need periods 😄 ...OCDs, sorry 😅